core/templates/domain/exploration/OutcomeObjectFactory.ts

```
@@ -28,7 +28,7 @@ import { ParamChangeBackendDict } from
   'domain/exploration/ParamChangeObjectFactory';
 
 export interface OutcomeBackendDict {
-  'dest': string;
+  'dest': string | null;
```

Why can this be `null`?

---

> Why can this be `null`?

All the nulls basically signify default values of these variables, I tried using empty string ' ' as default values but it results in frontend tests failing saying it expects them to be null instead of empty string ' '.

---

Removed null type from code.